marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/ingestion/bulkIngester.sjs

```diff
@@ -25,11 +25,38 @@ var input;         // jsonDocument*
 declareUpdate();
 
 const ingest = require("/data-hub/5/builtins/steps/ingestion/default/main.sjs");
-const consts = require('/data-hub/5/impl/consts.sjs');
+const consts = require("/data-hub/5/impl/consts.sjs");
+const HubUtils = require("/data-hub/5/impl/hub-utils.sjs");
 const state = fn.head(xdmp.fromJSON(endpointState));
 
 const work = fn.head(xdmp.fromJSON(workUnit));
 
+let collections = [];
```

Comment:
"const" is preferable to "let", as it makes it clear that the value will not be changing. So do this instead:

    const collections = work.collections != null ? work.collections.split(',') : [];

Reply:
Made the changes